### DISCUSSION TITLE
dax comments

### DISCUSSION THREAD

**Discussion by gazmanic** [DaxStudio/DaxStudio#795] [Ideas]
Hi,  

No doubt has been asked for already but could not find a reference.  Would be nice to be able to choose which comment character in the editor.  ie //  vs  --

I started using // but seems that -- maybe more widely used and takes up less visual space to my mind :)
Is there a block comment character in dax?   eg /*  */    This would be handy.

Thank you in advance
Gary

**Comment by marcosqlbi** [comment]
DAX supports both single line comments with // and multi line comments with /* and */
You're right, the Microsoft online documentation does not report the syntax.
It is certainly covered in our book (page 18): https://www.sqlbi.com/books/the-definitive-guide-to-dax-2nd-edition/

**Reply by gazmanic** [reply]
Thanks kindly Marco, have just completed the intro and tools course.  And probably should have done these before asking questions :}
